proto/buf.yaml

```
@@ -1,9 +1,11 @@
-# Generated by "buf config migrate-v1beta1". Edit as necessary, and
-# remove this comment when you're finished.
-#
-# This module represents the "proto" root found in
-# the previous configuration.
 version: v1
+name: buf.build/cosmos/ibc-go
```

This must match the schema registry which @tac0turtle created! 

```suggestion
name: buf.build/cosmos/ibc
```

---

If we move our proto files to cosmos/ibc. This will still be okay?

---

yea, its not tied to a repo.

---

I named ibc cause I wasn't sure if the proto files are specific to go or not. I can change if needed

---

I think ibc should be fine. There's been talks about potentially moving protos to spec repo anyways